packages/venia-concept/venia-upward.yml

```diff
@@ -68,97 +122,187 @@ isDevelopment:
   default:
     inline: false
 
+# A ProxyResolver object that passes a request to the backend Magento
+# server defined in the MAGENTO_BACKEND_URL environment variable.
+# An UPWARD server infers this object as a ProxyResolver due to the presence
+# of the 'target' property.
```

Comment:
I thought the upward server would know to use the `ProxyResolver` because of the "telltale" `proxy` on line 129. What does this mean then?

Reply:
'proxy' is just the name of the context object. As far as I know an UPWARD server only infers resolver types through its properties, not it's name.

See: https://github.com/magento-research/pwa-studio/tree/develop/packages/upward-spec#resolver-type-inference

Reply:
Great, thanks for the link!